validator/block.ts

```diff
@@ -5,14 +5,15 @@ import {blsSign} from "../src/stubs/bls";
 import {getDomain, slotToEpoch} from "../src/chain/helpers/stateTransitionHelpers";
 import {DOMAIN_BEACON_BLOCK, DOMAIN_RANDAO} from "./constants";
 import {getEmptyBlock} from "../src/chain/helpers/genesis";
+import {AbstractLogger} from "../src/logger";
```

Comment:
```javascript
import logger from "../src/logger";


logger.info("log");
```

It's singleton so you can just import and use it, no need to pass it as prop everywhere.

Reply:
@GregTheGreek think you missed this comment :)